document available host preflights types (#656)
replicatedhq/kurl.sh: src/markdown-pages/create-installer/host-preflights/host-services.md

---
path: "/docs/create-installer/host-preflights/host-services"
date: "2022-01-13"
weight: 31
linktitle: "Host Services"
title: "Host Services"
---
 

> The host services host preflight check can be used to detect and validate the status of certain host system services.

Reviewer: ```suggestion
You can use the host services host preflight check to detect and validate the status of certain host system services.
```
